alembic/versions/a44fb7695c10_add_reasons_foreign_key_to_correlations.py

```diff
@@ -0,0 +1,52 @@
+"""add reasons foreign key to correlations
+
+Revision ID: a44fb7695c10
+Revises: 373ef4569b93
+Create Date: 2016-05-10 09:02:08.516523
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = 'a44fb7695c10'
+down_revision = '373ef4569b93'
+
+
+def upgrade():
+    op.drop_index('correlations_signature_idx', table_name='correlations')
+    op.add_column(
+        'correlations', sa.Column('reason_id', sa.INTEGER(), nullable=True)
+    )
+    op.create_index(
+        'correlations_signature_idx',
+        'correlations',
+        [
+            'product_version_id',
+            'platform',
+            'key',
+            'date',
+            'signature_id',
+            'reason_id',
+        ],
+        unique=True,
+    )
+
+
+def downgrade():
+    op.drop_column(
```

Comment:
Don't you need to drop the index here as well? Or will it get removed automatically when removing one of its columns?